docs/0000-rfc-styleguide.md

```diff
@@ -0,0 +1,82 @@
+# RFC Name
+
+Request for Contribution (RFC) 0000 : _Style guide_
+
+## Code of conduct
+
+Please read and respect the [RFC Code of Conduct](https://github.com/umbraco/rfcs/blob/master/CODE_OF_CONDUCT.md)
+
+## Intended Audience
+
+The intended audience for this RFC is: 
+
+* documentation contributors
+* hq developers
+
+## Summary
+
+What kind of language and punctation rules should be set on the documentation.
+
+## Motivation
+
+To make sure that there is a consistent writing style and to improve readability we want to include some rules.  
+It will make first time contributors feel more confident on the approach to make documentation.
+
+## Detailed Design
+
+This is the main part of the RFC. How do you see this being implemented in Umbraco?
+
+### Usage of language
+
+While there has never been an official guide on the language being used.  The future tooling (docfx) allows us to have multi lingual documentation.  We would need to set guides for translators.  What words to we try to avoid.  What kind of tone do we keep when writng (e.g. friendly, ...).
+
+We try to avoid words like simply, just, ...  We should explain why we would like to avoid these word.  And create a list of words we try to avoid.  And have a list of words we need to avoid.
+
+### Layout rules
+
+We need to make sure, for consistency that we have simple layout rules.   
+As a documentation visitor, having the same punctuation and layout improves documentation. 
+
+### Method of enforcement 
+
+With the tooling like [Vale](https://errata-ai.github.io/vale/) we are able to enforce rules. Contributors should be able to checkout the documentation repository for the our documentation and check whether or not it can pass the enforced rules.
```

Comment:
It can feel petty for curators to point out some stylistic issues in a contribution, the automated tool can reduce that effort / be the referee it matters of style.